# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/wolfssl/libdir.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- wolfssl-5.8.0-stable/CMakeLists.txt.vanilla	2025-06-22 21:26:11.411460931 +0200
+++ wolfssl-5.8.0-stable/CMakeLists.txt	2025-06-22 21:32:18.721963800 +0200
@@ -2862,8 +2862,8 @@
     # Install the library
     install(TARGETS wolfssl
             EXPORT wolfssl-targets
-            LIBRARY DESTINATION lib
-            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib${LIBDIR_SUFFIX}
+            ARCHIVE DESTINATION lib${LIBDIR_SUFFIX}
             RUNTIME DESTINATION bin
             )
     # Install the headers
@@ -2895,7 +2895,7 @@
 
     set(prefix ${CMAKE_INSTALL_PREFIX})
     set(exec_prefix "\${prefix}")
-    set(libdir "\${exec_prefix}/lib")
+    set(libdir "${CMAKE_INSTALL_LIBDIR}")
     set(includedir "\${prefix}/include")
     set(VERSION ${PROJECT_VERSION})
 
